### ISSUE TITLE
Investigate why Harvard Dataverse is unable to harvest from International Potato Center and ICRISAT Research Data Repository

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse.harvard.edu#259]
Harvard Dataverse isn't able to harvest from two repositories that are using Dataverse - [International Potato Center](https://data.cipotato.org/) and [ICRISAT Research Data Repository](https://dataverse.icrisat.org/).

[Harvard Dataverse's managing harvesting client page](https://dataverse.harvard.edu/harvestclients.xhtml?dataverseId=1) has been reporting that trying to harvest from these two repositories fails, 

And when I try to create or edit these clients, I get an error that the installation's harvesting URLs, https://data.cipotato.org/oai and https://dataverse.icrisat.org/oai are invalid that Harvard Dataverse couldn't establish connection and receive a valid server response.

It looks like the repositories have OAI enabled, since https://data.cipotato.org/oai?verb=Identify and https://dataverse.icrisat.org/oai?verb=Identify seem to work.

**Comment by jggautier** [comment]
When I use Python's request library to get records from International Potato Center, I get a warning about an SSLCertVerificationError, which made me wonder if Dataverse doesn't want to harvest from International Potato Center because of that SSL certificate error.

I don't get this SSL certificate error when I try to use Python's request library to get records from ICRISAT Research Data Repository.

**Comment by pdurbin** [comment]
> records from International Potato Center today (July 10, 2024), I got a warning about an SSLCertVerificationError

Yes, I'm seeing similar errors from Java (server.log) when I show the detailed error: https://groups.google.com/g/dataverse-community/c/lpYLX4RkxO4/m/5KTRZQ-3AQAJ
